control-plane/csi-driver/controller/src/pvwatcher.rs

```diff
@@ -0,0 +1,146 @@
+use anyhow::Result;
+use futures::TryStreamExt;
+use k8s_openapi::api::core::v1::PersistentVolume;
+
+use kube::{
+    api::{Api, ListParams},
+    runtime::{watcher, WatchStreamExt},
+    Client,
+};
+use tracing::{debug, error, info};
+use uuid::Uuid;
+
+use crate::IoEngineApiClient;
+
+const MAYASTOR_PLUGIN: &str = "io.openebs.csi-mayastor";
+const BOLT_PLUGIN: &str = "com.datacore.bolt";
```

Comment:
Please remove the `BOLT_PLUGIN`, is not meant to be here.

Reply:
Incorporated